lib/fizzy/execute.cpp

```diff
@@ -129,7 +140,11 @@ inline uint64_t popcnt64(uint64_t value) noexcept
 
 Instance instantiate(const Module& module)
 {
-    return {module};
+    // FIXME: set pages from proper section
+    constexpr unsigned pages = 1;
+    // NOTE: fill it with zeroes
+    bytes memory(pages * pagesize, 0);
+    return {module, std::move(memory)};
```

Comment:
I think better without `move` or even just construct inline
```suggestion
    return {module, bytes(pages * pagesize)};
```

Reply:
I'd prefer to keep the explicit `memory` name for now.